pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -1012,17 +1047,17 @@ func (qjm *XController) getAggregatedAvailableResourcesPriority(unallocatedClust
 }
 
 func (qjm *XController) chooseAgent(qj *arbv1.AppWrapper) string {
-	
+
 	if qjm.serverOption.ExternalDispatch {
 		clusters := qj.Spec.SchedSpec.ClusterScheduling.Clusters
 		var agentId = ""
-        apath := path.Dir(qjm.agentList[0]) 
+		apath := path.Dir(qjm.agentList[0])
 		var agentIdList = make([]string, len(clusters))
-		clustersProvided := false  // assume clusters not provided
+		clustersProvided := false // assume clusters not provided
 		for _, clusterRef := range clusters {
 			if clusterRef.Name != "" {
 				clustersProvided = true
-				agentIdList = append(agentIdList, apath+"/"+clusterRef.Name )
+				agentIdList = append(agentIdList, apath+"/"+clusterRef.Name)
```

Comment:
Will this logic work with the original implementation of the cluster list selection?  The original implementation that does not use KubeStellar.

Reply:
Yes, this code is only active when running with externalDispatch=true:

	if qjm.serverOption.ExternalDispatch { ... return agentId }

Right beyond this block is the original code which I did not change